packages/api/src/controllers/did.js

```diff
@@ -0,0 +1,9 @@
+import Router from "express/lib/router";
+
+const app = Router();
+
+app.get("/", (_, res) => {
+  res.json({ did: process.env.UCAN_DID || "" });
```

Comment:
1. I think the env variable name is not correct. In your PR https://github.com/livepeer/livepeer-infra/pull/1163, you defined it as `LP_API_DID`, here you use `UCAN_DID`. Could you double-check?

Also, please check how the environment variables are resolved in Livepeer Studio. I see that the `LP_API_` prefix is parsed somehow as flags, so please check if we should pass it somehow via flags to be consistent with other env variables.

2. I think that if the variable is not defined we should not return an empty string, but rather the specific HTTP Error code, I think `501` in this case.

3. I'd consider some simple validation if the env variable is in the correct format `did:key:...`.

Reply:
You should actually read that env from the parsed `req.config.did` instead! With the corresponding flag declaration in `parse-cli.ts` file. Those can also come from env vars (if you're curious check yargs usage), and in fact all of them do in our production deployments.